### PR TITLE
Sparse live tests

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: eng/pipelines/templates/jobs/live.tests.yml [hunk 1]
```diff
@@ -77,6 +77,13 @@ jobs:
       container: $[ variables['Container'] ]
 
     steps:
+      - ${{ if not(contains(variables['Build.DefinitionName'], '-pr - ')) }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+              Paths:
+                - "/*"
```
Comment (benbp) [inline eng/pipelines/templates/jobs/live.tests.yml:84]: You could probably narrow this down further but it may be diminishing returns.

**Review by benbp — APPROVED** [review]

File: eng/pipelines/templates/jobs/live.tests.yml (hunk 1 above)
Comment (pakrym) [reply eng/pipelines/templates/jobs/live.tests.yml:84]: Yeah, I think the largest win is coming from SessionRecords. When you go deeper project-project dependencies start to matter and things get messy.

**Pull request merged**